examples/.experimental/next-app-dir/src/app/post-example/page.tsx

```diff
@@ -0,0 +1,42 @@
+import { api } from '~/trpc/server-http';
+
+async function action(fd: FormData) {
+  'use server';
+
+  // create the post
+  await api.createPost.mutate({
+    title: fd.get('title') as string,
+    content: fd.get('content') as string,
+  });
+
+  // revalidate the latest post
+  await api.getLatestPost.revalidate();
+}
```

Comment:
Could we do 

```ts
await api.createPost.mutate(fd);
```

?

Reply:
i don't think so - we do the normalization for server actions but i don't think we do it for mutations?

i can look into it in #4567 👍🏼